app/assets/stylesheets/forever_style_guide/modules/_fa-feature_bullet.scss

```diff
@@ -23,6 +23,14 @@
     color: color('accent');
   }
 
+  &.fa-feature_bullet-services .fa-feature_bullet-circle {
+    color: color('services');
+  }
+
+  &.fa-feature_bullet-ambassador .fa-feature_bullet-circle {
```

Comment:
For better flexibility, we should dynamically generate these based on core colors, the same way we do color_blocks et al.